ampoule/pool.py

```diff
@@ -148,18 +148,24 @@ def _pruneProcess(self, child):
         self._calls.pop(child, None)
         self._finishCallbacks.pop(child, None)
 
+    def fatal(self, reason, child):
```

Comment:
Is the idea here that this is a hook that could be overridden in subclasses?  I intentionally avoided exposing it as a method to prevent that 😉 .

Can you make this a parameter to the constructor instead?